megarules.py

```diff
@@ -445,18 +466,31 @@ def mega_rules(AOI_name, coordinates, slcp_rule, lar_rule, ifg_rule, cod_rule, s
     lar = ctx['LAR_processing']
     cod = ctx['COD_processing']
 
-    ifg_workflow = ctx['ifg_workflow']
-    slcp_workflow= ctx['slcp_workflow']
-    lar_workflow = ctx['lar_workflow']
-    cod_workflow = ctx['cod_workflow']
+    ifg_params = {}
```

Comment:
The following code are additional params added (via context) to deal with additional inputs (az,rn looks) for:
1. LAR
2.CODs

For SLCPs, a value (7,3)  is used as input as multilooking is not actually applied in SLCP.